### PR TITLE
squared difference op

### PR DESCRIPTION
**Pull request by jd-bartlett96**
Implemented squared difference op + test

### CODE REVIEW

File: plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/squared_difference.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+};
+
+const std::vector<std::vector<std::size_t>> inputShapes = {
+    {std::vector<std::size_t>({1, 30}), std::vector<std::size_t>({1, 30})}};
+
+INSTANTIATE_TEST_CASE_P(CompareWithRefs, SquaredDifferenceLayerTest,
+                        ::testing::Combine(::testing::ValuesIn(inputPrecisions), ::testing::Values(inputShapes),
+                                           // ::testing::ValuesIn(netPrecisions),
+                                           ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)),
+                        // ::testing::ValuesIn(std::vector<int>{1, 2}),  // TODO
```
Comment (tzerrell) [inline plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/squared_difference.cpp:30]: No need to keep these 2 commented-out lines

File: plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/squared_difference.cpp [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+using LayerTestsDefinitions::SquaredDifferenceLayerTest;
+
+namespace {
+const std::vector<InferenceEngine::Precision> inputPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
```
Comment (tzerrell) [inline plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/squared_difference.cpp:21]: Only need one precision for the style of test we're using (I think I've mostly been calling it netPrecision, although with this style of test they're the same thing)

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/squared_difference.cpp [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("squareddifference", [](const Context& ctx) {
+  IE_ASSERT(ctx.operands.size() == 2);
+  auto A = ctx.operands.at(0);
+  auto B = ctx.operands.at(1);
+  return edsl::make_tuple(op::square(A - B));
+  // return edsl::make_tuple(1);
```
Comment (tzerrell) [inline plaidml/bridge/openvino/src/plaidml_plugin/ops/squared_difference.cpp:22]: No need to keep this line

**Comment by tzerrell** [comment]
You'll presumably want to update `vendor/openvino` to get the changes from https://github.com/plaidml/openvino/pull/13

File: vendor/openvino/workspace.bzl [hunk 4]
```diff
@@ -25,8 +25,8 @@ def openvino_workspace():
 
     http_archive(
         name = "openvino",
-        sha256 = "3831e7ef94dd5126dd1dba30e9fde181d87f0a264d406181fec08e1bac862d4f",
-        strip_prefix = "openvino-455b635a1cc68ef5cc5c35d4ea0fb16ce13923f9",
-        url = "https://github.com/PlaidML/openvino/archive/455b635a1cc68ef5cc5c35d4ea0fb16ce13923f9.zip",
+        sha256 = "4f12aaf04164ea516ea7915d21a7d9a9f982cc63b6743818a4faed9efdafa6ae",
+        strip_prefix = "openvino-daf3d70b8f160869c094a151bf83ee3f6e2317be",
```
Comment (tzerrell) [inline vendor/openvino/workspace.bzl:29]: I don't think this is the right hash. See: https://github.com/plaidml/openvino/commits/plaidml

Comment (jd-bartlett96) [reply vendor/openvino/workspace.bzl]: Should be good now

**Pull request merged**